spec/dummy/app/models/with_paperclip.rb

```diff
@@ -1,5 +1,3 @@
-require "paperclip/pdf_cover"
```

Comment:
I think it should be loaded by the PdfCover module, not by test (and manually by the Rails application that uses the gem).